testing/rdiffbackupdeletetest.py

```diff
@@ -103,6 +108,13 @@ def test_delete_with_directory(self):
         rdiff_backup(1, 1, self.repo, None, extra_options=b"--verify")
         self.assertNotFound(b'tmp')
 
+    def test_delete_with_directory_with_perm(self):

```

Comment:
short description of what is being tested doesn't hurt...

Reply:
done